stable/redis/README.md

```diff
@@ -43,36 +49,87 @@ The command removes all the Kubernetes components associated with the chart and
-| `metrics.enabled`             | Start a side-car prometheus exporter              | `false`                      |
-| `metrics.image`               | Exporter image                                    | `oliver006/redis_exporter`   |
-| `metrics.imageTag`            | Exporter image                                    | `v0.11`                      |
-| `metrics.imagePullPolicy`     | Exporter image pull policy                        | `IfNotPresent`               |
-| `metrics.resources`           | Exporter resource requests/limit                  | Memory: `256Mi`, CPU: `100m` |
-| `nodeSelector`                | Node labels for pod assignment                    | {}                           |
-| `tolerations`                 | Toleration labels for pod assignment              | []                           |
-| `networkPolicy.enabled`       | Enable NetworkPolicy                              | `false`                      |
-| `networkPolicy.allowExternal` | Don't require client label for connections        | `true`                       |
-| `service.annotations`         | annotations for redis service                     | {}                           |
-| `service.loadBalancerIP`      | loadBalancerIP if service type is `LoadBalancer`  | ``                           |
-| `securityContext.enabled`     | Enable security context                           | `true`                       |
+| Parameter                                  | Description                                                                                                    | Default                              |               |
+|--------------------------------------------|----------------------------------------------------------------------------------------------------------------|--------------------------------------|---------------|
+| `image`                                    | Redis image                                                                                                    | `bitnami/redis`                      |               |
+| `imagePullPolicy`                          | Image pull policy                                                                                              | `IfNotPresent`                       |               |
+| `imageTag`                                 | Image tag                                                                                                      | `{VERSION}`                          |               |
+| `imageCredentials.registry`                | Registry where the image is located                                                                            | `docker.io`                          |               |
+| `imageCredentials.username`                | Username for accessing the image registry                                                                      | `nil`                                |               |
```

Comment:
This is unconventional. It's more accepted to point directly to a Secret instead.

Reply:
This is based on this official example: https://github.com/kubernetes/helm/blob/master/docs/charts_tips_and_tricks.md#creating-image-pull-secrets

Reply:
I'd agree with @timstoop that we should point to the secret

Reply:
@prydonius @timstoop changed it to secrets